ci/scale-run.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Read variables
+source ovn-scale.conf
+
+OVS_REPO=$1
+OVS_BRANCH=$2
+
+mkdir -p $OVN_DOCKER_ROOT
+pushd $OVN_DOCKER_ROOT
+
+# Install OVN Scale Test
+if [ ! -d $OVN_SCALE_REPO_NAME ]; then
+    git clone $OVN_SCALE_REPO
+    pushd $OVN_SCALE_REPO_NAME
+    git fetch
+    git checkout $OVN_SCALE_BRANCH
+    popd
+fi
+
+# Build the docker containers
+pushd $OVN_SCALE_REPO_NAME
+cd ansible/docker
+make
+popd
+
+# Deploy the containers
+pushd $OVN_SCALE_REPO_NAME
+sudo /usr/local/bin/ansible-playbook -i $OVN_DOCKER_HOSTS ansible/site.yml -e @$OVN_DOCKER_VARS \
```

Comment:
After executing the asnbile command, can we add some sanity check to ensure all emulated hypervisor are connected to the southbound db, e.g., by "docker exec southbound-database ovn-sbctl show"?